### ISSUE TITLE
Support Grazie in IntelliJ 2020.1 release

### ISSUE THREAD

**Issue by ahus1** [asciidoctor/asciidoctor-intellij-plugin#408]
This implements the interface exposed in the latest development of Grazie

```kotlin
  /**
   * Determine if this strategy enabled by default.
   *
   * @return true if enabled else false
   */
  @JvmDefault
  fun isEnabledByDefault(): Boolean = !GraziePlugin.isBundled || ApplicationManager.getApplication()?.isUnitTestMode.orTrue()
```

**Comment by ahus1** [comment]
@TanVD - I added the necessary changes in a downwards compatible way. Version 0.30.39 of the plugin is available. I'll keep this issue open until the next EAP version is available.

**Comment by TanVD** [comment]
New strategy was released with 201.5985.32:

Note, that you'll need to:
* override `isEnabledByDefault` with `true`
* strategies now should include comments -- so you should at least add `|| element is PsiComment` 
  to `isMyContextRoot`
* you can specify domain of text blocks (is it comment, documentation or plain-text)

**Comment by ahus1** [comment]
I've released verison 0.30.40 with an added `|| element is PsiComment`.

For now I'll not implement getContextRootTextDomain to binary compatibility with 2019.3 and leave it for a later release.

I'm closing this ticket as these changes will work for 2020.1 AFAIK. Please comment if I missed something.

Thanks for pinging me on this API change!
